From 40a7d10197396a011ae07bcec587da35c9e548f5 Mon Sep 17 00:00:00 2001
From: kostap <kostap@marvell.com>
Date: Thu, 19 Jun 2014 09:00:01 +0300
Subject: [PATCH 1721/1825] fix: msys: Fix the Ethernet port offset value

https://github.com/MISL-EBU-System-SW/misl-windriver.git linux-3.4.69-14t2-read
commit afe13f7289f62ec7bbd22ec707d9a416a14b1c99

	- Fix the offset value for Ethernet port registers
	  from 0x40000 to 0x4000 per port

Change-Id: I0e9023c703a9b64cb50059b4bbe7d702342d68d9
Signed-off-by: kostap <kostap@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/8597
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
Tested-by: Omri Itach <omrii@marvell.com>
Signed-off-by: Zhong Hongbo <hongbo.zhong@windriver.com>
---
 .../mach-msys/msys_family/ctrlEnv/mvCtrlEnvSpec.h  |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-msys/msys_family/ctrlEnv/mvCtrlEnvSpec.h b/arch/arm/mach-msys/msys_family/ctrlEnv/mvCtrlEnvSpec.h
index 2db8944..d02b973 100644
--- a/arch/arm/mach-msys/msys_family/ctrlEnv/mvCtrlEnvSpec.h
+++ b/arch/arm/mach-msys/msys_family/ctrlEnv/mvCtrlEnvSpec.h
@@ -105,7 +105,7 @@ extern "C" {
 #define MV_PMU_NFABRIC_UNIT_SERV_OFFSET		(0x22000)
 #define MV_CPU_PMU_UNIT_SERV_OFFSET(cpu)	(0x22100 + (cpu) * 0x100)
 #define MV_ETH_BASE_ADDR			(0x70000)
-#define MV_ETH_REGS_OFFSET(port)		(MV_ETH_BASE_ADDR + (port)* 0x40000 )
+#define MV_ETH_REGS_OFFSET(port)		(MV_ETH_BASE_ADDR + (port) * 0x4000)
 #define MV_PEX_IF_REGS_OFFSET(pexIf)\
 			(pexIf < 8 ? (0x40000 + ((pexIf) / 4) * 0x40000 + ((pexIf) % 4) * 0x4000)\
 	: (0X42000 + ((pexIf) % 8) * 0x40000))
-- 
1.7.5.4

